composer/loggers/logger_hparams.py

```diff
@@ -104,10 +95,12 @@ class WandBLoggerHparams(LoggerDestinationHparams):
             If not specified, the :attr:`~composer.loggers.logger.Logger.run_name` will be used.
         entity (str, optional): WandB entity name.
         tags (str, optional): WandB tags, comma-separated.
-        log_artifacts (bool, optional): See
-            :class:`~composer.loggers.wandb_logger.WandBLogger`.
+        config (Dict[str, Any], optional): WandB run configuration.
+        flatten_config (bool, optional): Whether to flatten the run config. (default: ``False``)
```

Comment:
This docstring has `flatten_config`, but the method signature has `flatten_hparams`. I prefer `flatten_config`, personally.